tests/unit/fetcher-jsonld-test.js

```diff
@@ -0,0 +1,40 @@
+import * as rdf from '../../src';
+
+const nock = require('nock')
+import chai from 'chai'
+
+const {expect} = chai
+
+describe('Given a JSON-LD resource', () => {
+
+    const uri = "http://localhost/jsonld#it"
+
+    beforeEach(() => {
+        const docContents = `
+        {
+            "@id": ${uri},
```

Comment:
There's a small typo here:
```suggestion
            "@id": "${uri}",
```
I haven't run the test, but if it passes now, then maybe the issue you were facing was #408? 
Note also that another bug caused jsonld errors to be suppressed: #435 

I've added some annotations to the relevant commit https://github.com/linkeddata/rdflib.js/commit/16daad7d68d805aa29e8e01bb4a16d77b28b6afe#r41313390

Reply:
Or is there a separate issue related to fetching, like in https://github.com/linkeddata/rdflib.js/issues/343?

Reply:
Good catch! Fixed the syntax. But test is still failing for the same reason. I think the issue is in the fetching / response handling.

There is just no default handler for JSON-LD?

```
const defaultHandlers = {
  RDFXMLHandler, XHTMLHandler, XMLHandler, HTMLHandler, TextHandler, N3Handler
}
```